### DISCUSSION TITLE
Express in ProseObject format - extend to other materials

### DISCUSSION THREAD

**Discussion by HazardJ** [Open-Weights/Definition#2] [General]
This is a great initiative!  Following up on prior conversations, I wonder if this might be a kernel for moving legal to a "source" format along the lines of www.commonaccord.org.  In particular, to start with these Neural Net Weights, and extend into licenses and terms relating to AI/LLMs, and beyond.

The "source" thought is that we can apply the notions of modularity and versioning to legal documents, creating models that can be used by reference and granular overriding.
  
Some Licenses in ProseObject format:  http://www.commonaccord.org/i.php?v=s&f=S/Index/DataSharing/License.md

**Comment by heathermeeker** [comment]
@HazardJ how does one go about that coding?

**Reply by HazardJ** [reply]
As a demo, I forked the repo and added ProseObject versions of the Definition and the License in a subfolder.  The demo is available at http://www.commonaccord.org/i.php?v=l&f=G/Open-Weights/Pr0b/. 
The forked repo is at https://github.com/CommonAccord/Open-Weights--Definition.  I could do a pull-request if this looks useful.

**Reply by HazardJ** [reply]
I have given rather arbitrary meaningful (semantic) names to the subsections of the Definition and License.   This demonstrates semantic "composability".  :)
E.g., the "Scope" subsection of the License can be fetched with: http://www.commonaccord.org/i.php?v=d&f=G/Open-Weights/Pr0b/License.md&k=Scope.r00t
